source/_integrations/ecobee.markdown

```diff
@@ -18,41 +18,38 @@ ha_domain: ecobee
 
 The `ecobee` integration lets you control and view sensor data from [ecobee](https://ecobee.com) thermostats.
 
-## Preliminary Step
+## Preliminary Steps
 
 You will need to obtain an API key from ecobee's [developer site](https://www.ecobee.com/developers/) to use this integration. To get the key, your thermostat must be registered on ecobee's website (which you likely would have already done while installing your thermostat). Once you have done that, perform the following steps.
 
-1. Click on the **Become a developer** link on the [developer site](https://www.ecobee.com/developers/).
-2. Log in with your ecobee credentials.
-3. Accept the SDK agreement.
-4. Fill in the fields.
-5. Click **save**.
+1. Click on the **Become a developer** link on the [developer site](https://www.ecobee.com/developers/)
+2. Log in with your ecobee credentials
+3. Accept the SDK agreement
+4. Fill in the fields
+5. Click **save**
 
```

Comment:
Please revert this change. Lists that contain a sentence must end with a period. If one item in the list adheres to this, all items get a full stop.

See: <https://docs.microsoft.com/en-us/style-guide/punctuation/periods>

Reply:
Good to know!